src/app/pages/FrontPage/index.jsx

```diff
@@ -32,6 +43,20 @@ const FrontPageMostReadSection = styled(MostReadSection)`
   }
 `;
 
+const StyledRadioScheduleContainer = styled(RadioScheduleContainer)`
```

Comment:
Sorry for being a bit late to the review, but would it be possible to export these styles from the Radio Schedule folder? So we can't import them in both, FrontPage and IDX pages? 
Also, we have been trying to keep the page containers without styling definitions to reduce their size.

Reply:
Hi Denis, do you have something along these lines in mind? https://github.com/bbc/simorgh/pull/7260

Reply:
That's exactly what I meant 👌🏻